### ISSUE TITLE
simple_server.py example does not work

### ISSUE THREAD

**Issue by Avernial** [tango-controls/pytango#183]
simple_server.py example not working in python3.
```
Traceback (most recent call last):
  File "/home/aver/workspace/snippets/test_wait.py", line 38, in <module>
    server.register_object(my_object, "my_object")
  File "/usr/lib64/python3.4/site-packages/pytango-9.2.2-py3.4-linux-x86_64.egg/tango/tango_object.py", line 552, in register_object
    member_filter=member_filter)
  File "/usr/lib64/python3.4/site-packages/pytango-9.2.2-py3.4-linux-x86_64.egg/tango/tango_object.py", line 158, in create_tango_class
    cmd = types.MethodType(cmd, None, DeviceDispatcher)
TypeError: method expected 2 arguments, got 3
```
The problem is that in python 3 `types.MethodType` accept only two arguments:
```
Init signature: MethodType()
Docstring:     
method(function, instance)

Create a bound instance method object.
```

Removing `None` from arguments in function `create_tango_class` solves this problem.

```
cmd = types.MethodType(cmd, DeviceDispatcher)
```

**Comment by vxgmichel** [comment]
Thanks for the report! It will be fixed after PR #184 is merged.

**Comment by vxgmichel** [comment]
Merged, the fix will become visible with v9.2.3.
